backend-go/api/codepair/v1/models/model_hello_request.go

```diff
@@ -0,0 +1,7 @@
+package models
+
+type HelloRequest struct {
+
+	// New nickname to say hello
+	Nickname string `json:"nickname"`
+}
```

Comment:
How about writing a Request, Response, and Validator of the same model (`Hello`) in the same file? It seems unnecessary to write the Request and Response in different files.

Maybe it would be better to write them together in the router file.

Reply:
I didn’t realize these files were auto-generated by OpenAPI generation. If that’s the case, I think it’s great!